src/components/pages/dashboard-pages/AddClass.jsx

```diff
@@ -39,17 +48,14 @@ const AddClass = () => {
           [e.target.name]: e.target.value,
         },
       });
-    } else if (e.target.name === 'date' || e.target.name === 'startTime')
+    } else if (e.target.name === 'date' || e.target.name === 'startTime') {
+      const newAvailableTime = [...formData.availableTime];
```

Comment:
Hooray! Now we can create classes and add multiple available time!

Thank you,Valya!